src/app/components/osd/osd.component.ts

```diff
@@ -112,12 +114,14 @@ export class OsdComponent implements AfterViewInit, OnDestroy {
   @Input() set page(v: number) {
     if (v !== this._page) {
       this._page = v;
-      this.pageChange.next(this._page);
+      this.pageChange.next(this._page + 1);
```

Comment:
Why the `+1`?

Reply:
@saramaenza can answer to @szenzaro? Why do you need the `+1`?